extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1689,7 +1689,10 @@ export class AmpStoryPage extends AMP.BaseElement {
    */
   setPageDescription_() {
     if (this.isBotUserAgent_) {
-      this.renderPageDescription_();
+      renderPageDescription(
+        this,
+        this.getMediaBySelector_(Selectors.ALL_AMP_VIDEO, true)
```

Comment:
Second parameter `includeHiddenMedia` should be `false`. This is meant to be used by publishers to play different videos based on some CSS media queries, but they're supposed to be different versions of the same one, ie: one landscape, one portrait.
If we load the captions from all the videos we'll likely end up with duplicated content on the page.

Reply:
Interestingly this fails if I don't set it. Could there be a timing issue?

Reply:
Yes I think it would work if we called this after the layoutCallback (right now it's called from the buildCallback). But this feature isn't documented or used so we're good for now, we can fix later if it ever becomes an issue.